src/core/gui/dialog/SettingsDialog.cpp

```diff
@@ -16,16 +16,16 @@
 #include "gui/CreatePreviewImage.h"              // for createPreviewImage
 #include "gui/MainWindow.h"                      // for MainWindow
 #include "gui/XournalView.h"                     // for XournalView
-#include "gui/widgets/ZoomCallib.h"              // for zoomcallib_new, zoom...
-#include "model/PageType.h"                      // for PageType
-#include "util/Color.h"                          // for GdkRGBA_to_argb, rgb...
-#include "util/PathUtil.h"                       // for fromGFile, toGFilename
-#include "util/StringUtils.h"                    // for StringUtils
-#include "util/Util.h"                           // for systemWithMessage
-#include "util/gtk4_helper.h"                    //
-#include "util/i18n.h"                           // for _
-#include "util/raii/CairoWrappers.h"             // for CairoSurfaceSPtr
-#include "util/safe_casts.h"                     // for round_cast
+#include "gui/toolbarMenubar/ToolMenuHandler.h"  // for ToolMenuHandler
+#include "gui/toolbarMenubar/icon/ColorIcon.h"
+#include "gui/toolbarMenubar/model/ColorPalette.h"  // for Palette
+#include "gui/widgets/ZoomCallib.h"  // for zoomcallib_new, zoom...#include "model/PageType.h"                         // for PageType
+#include "model/PageType.h"   // for PageType
+#include "util/Color.h"       // for GdkRGBA_to_argb, rgb...
+#include "util/PathUtil.h"    // for fromGFile, toGFilename
+#include "util/Util.h"        // for systemWithMessage#include "util/gtk4_helper.h"                       //
+#include "util/i18n.h"        // for _#include "util/raii/CairoWrappers.h"                // for CairoSurfaceSPtr
+#include "util/safe_casts.h"  // for round_cast
```

Comment:
Many includes moved into the comment of the line above.

Reply:
:facepalm: Don't know how that happend. But thank you very much for catching it!